libraries/engage/components/MessageBar/index.jsx

```diff
@@ -22,19 +22,25 @@ const MessageBar = memo(({ messages, classNames }) => (
         message,
         messageParams = null,
         translated,
+        icon: Icon = null,
       } = item;
 
       const messageOutput = !translated ? i18n.text(message, messageParams) : message;
 
       return (
         <div
           key={`${type}-${message}`}
-          className={css(classNames.message, styles[type])}
+          className={css(
+            classNames.message,
+            styles[type],
+            Icon ? styles.withIcon : null
+          )}
         >
+          {Icon && <Icon className={css(classNames.icon, styles.icon).toString()} /> }
           <span className={styles.srOnly}>
             {`${i18n.text(`cart.message_type_${type}`)}: ${messageOutput}`}
           </span>
-          <span aria-hidden>
+          <span aria-hidden className={Icon ? styles.messageToIcon : undefined}>
```

Comment:
You don't need to use `undefined` here. The common "react"-way is, to use `null`. This looks a little weird ;) React considers `null` as "not set". Please change this.